src/App.tsx

```diff
@@ -85,8 +86,8 @@ export default function App(): JSX.Element {
                   <Route exact path="/">
                     <h1>Etusivu</h1>
                   </Route>
-                  <Route
-                    id="resource-route"
+                  <PrivateRoute
+                    id="target-route"
```

Comment:
Will this change resource back to target on purpose? Maybe we need to first rebase this to master?

Reply:
I'll fix that one